ocean-client/src/vault-maxi.ts

```diff
@@ -109,10 +114,16 @@ export async function main(event: maxiEvent): Promise<Object> {
         if (0 < usedCollateralRatio && usedCollateralRatio < settings.minCollateralRatio) {
             result = await program.decreaseExposure(vault, telegram)
             exposureChanged = true
+            vault= await program.getVault() as LoanVaultActive
         } else {
             result = true
             exposureChanged = await program.checkAndDoReinvest(vault, telegram)
-            if(!exposureChanged) {
+            console.log("got "+(context.getRemainingTimeInMillis()/1000).toFixed(1)+" sec left after reinvest")
+            if(exposureChanged){
+                vault= await program.getVault() as LoanVaultActive 
+            }
+            if(context.getRemainingTimeInMillis() > 300*1000) {//min 5 minutes for action
```

Comment:
do we want to define a "remainingTimeNeeded" - maybe we might want to change it in future => just need to change it in one place instead of more.